core/scripts/run-tests.sh

```diff
@@ -3,6 +3,59 @@
 /**
  * @file
  * This script runs Backdrop tests from command line.
+ *
+ *
+ *  Run Backdrop tests from the shell.
+ *
+ *  Usage:        {$args['script']} [OPTIONS] <testClassNames>
+ *  Example:      {$args['script']} --url="http://backdrop" BootstrapPageCacheTestCase
```

Comment:
Let's replace `{$args['script']}` with `./run-tests.sh` here. When executed through the shell, this would tell you the name of the script that is being executed, but for static documentation we should just use the current file name.